website/routes.py

```diff
@@ -330,6 +330,16 @@ def make_url_map(app):
             'get', website_views.get_dashboard, json_renderer),
     ], prefix='/api/v1')
 
+    process_rules(app, [
+        Rule('/dashboard/confirmed_emails/', 'get', auth_views.confirm_user_get, json_renderer),
+        Rule('/dashboard/confirmed_emails/', 'put', auth_views.add_confirmed_email, json_renderer)
+
+    ], prefix='/api/v1')
+
+    process_rules(app, [
+        Rule('/dashboard/remove_confirmed_emails/', 'put', auth_views.confirm_email_remove, json_renderer)
```

Comment:
The only reason not to is that `DELETE`s don't accept JSON bodies, but instead do things one at a time via their ID. So if you're trying to do multiple removals at once, that's tricky. Otherwise definitely.